commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceTestUtil.java

```diff
@@ -91,6 +126,38 @@ public static CommerceOrder addB2CCommerceOrder(
 			commerceAccount.getCommerceAccountId(), commerceCurrencyId);
 	}
 
+	public static CommerceOrder addB2CCommerceOrder(
+			long groupId, long userId, long commerceCurrencyId,
+			long siteGroupId)
+		throws Exception {
+
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext(groupId);
+
+		if (userId == 0) {
+			userId = serviceContext.getUserId();
+		}
+
+		if (commerceCurrencyId == 0) {
+			CommerceCurrency commerceCurrency =
+				CommerceCurrencyTestUtil.addCommerceCurrency(groupId);
+
+			commerceCurrencyId = commerceCurrency.getCommerceCurrencyId();
+		}
+
+		CommerceAccount commerceAccount =
+			CommerceAccountLocalServiceUtil.addPersonalCommerceAccount(
+				userId, StringPool.BLANK, StringPool.BLANK, serviceContext);
+
+		long commerceChannelGroupId =
+			CommerceChannelLocalServiceUtil.
+				getCommerceChannelGroupIdBySiteGroupId(siteGroupId);
```

Comment:
_Ideally_, these variable declarations would be swapped, since `commerceChannelGroupId` is used before `commerceAccount` in the lines below. See https://docs.google.com/document/d/188YFP0V4n9WJIPYZL3jGBRBvEsvBNmHOU97Fdd_g6Ew/edit#heading=h.dnltur49wl4o for the general rule.